packages/blade/scripts/generateBundleSizeInfo.js

```diff
@@ -0,0 +1,95 @@
+const fs = require('fs');
+const path = require('path');
+const babelParser = require('@babel/parser');
+const traverse = require('@babel/traverse').default;
+const execa = require('execa');
+
+const main = () => {
+  // Get all the components name exported from the bundle and add them to the size-limit configuration
+  const fileContent = fs.readFileSync(
+    path.resolve(__dirname, '../build/lib/web/production/components/index.js'),
+    'utf8',
+  );
+
+  // Parse the file content to generate an AST
+  const ast = babelParser.parse(fileContent, {
+    sourceType: 'module',
+  });
+
+  // Arrays to store size-limit stats and component names to exclude
+  const sizes = [];
+  const excludedComponents = [
+    'useTheme',
+    'BladeProvider',
```

Comment:
BladeProvider should not be excluded, it's an important entry point for blade and if we accidentally import some third party lib here or mess up treeshaking on this component could affect all other components.

Reply:
I'll include it.